core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -10,6 +10,20 @@
  */
 public class MerlinsBeard {
 
+    public enum NetworkType {
```

Comment:
@blundell Haven't find a mention to enum in the post, can you please elaborate why do not use enum here?